### DISCUSSION TITLE
How to change the Environment light of an imported USD scene?

### DISCUSSION THREAD

**Discussion by IRONIC3D** [GPUOpen-LibrariesAndSDKs/BlenderUSDHydraAddon#204] [Q&A]
Hey Everyone.

So I'm importing a USD scene, if I'm rendering the scene alone then we have the typical white environment which I cannot figure out how to change. If I merge with a Blender Scene I can make use of the lights within that scene, however they are added on top of the white environment (Changing the render mode to Direct Illumination doesn't change that).

What I am after is using the Sky and Sun from ProRender. In the environment section I can only see the Sycle background, and attaching a sky and sun in there doesn't work because ProRender doesn't recognize it. I can see in the custom parameters in that panel that the default environment is set to IBL with an empty image, and the sky and sun is left empty. There is no way for me to input any parameters or change that behavior through the API.

Any idea on how to achieve this?

**Comment by DagerD** [comment]
Hi.

Right now we only support filtering by fullmatch. We will discuss how we can improve it.

Added as BLEN-54.

**Reply by DagerD** [reply]
See #243
